blog/tutorial/01-index.md

```diff
@@ -24,6 +24,17 @@ This tutorial uses [Starport](https://github.com/tendermint/starport) v0.17.0. S
 curl https://get.starport.network/starport@v0.17.0! | bash
 ```
 
+
+In case the above command does not work, use the following:
+
+```
+curl https://get.starport.network/starport > starport.sh
```

Comment:
```suggestion
curl https://get.starport.network/starport@0.17.0! > starport.sh
```

Reply:
this tutorial is tested for Starport v0.17.0, so let's include that version here too